docs/Makefile

```diff
@@ -73,6 +73,10 @@ singlehtml:
 	@echo
 	@echo "Build finished. The HTML page is in $(BUILDDIR)/singlehtml."
 
+.PHONY: livehtml
+livehtml:
```

Comment:
How about we tie it together and add a comment?

```suggestion
# doc-html-live target starts a server mode for sphinx that watches the doc
# directory and will update if it sees changes. the doc-html target produces
# static html files.
.PHONY: doc-html-live
doc-html-live:
```

Reply:
I added the target into the help target. That way it will be listed when simply running `make` without any target.